boards/pimoroni-plasma-2040/README.md

```diff
@@ -0,0 +1,100 @@
+# [pimoroni-plasma-2040] - Board Support for the [Pimoroni Plasma 2040]
+
+You should include this crate if you are writing code that you want to run on
+a [Pimoroni Plasma 2040] - Swathe everything in rainbows with this all-in-one, USB-C powered controller for WS2812/Neopixel and APA102/Dotstar addressable LED strip.
+
+This crate includes the [rp2040-hal], but also configures each pin of the
+RP2040 chip according to how it is connected up on the Pimoroni Plasma 2040.
+
+[Pimoroni Plasma 2040]: https://shop.pimoroni.com/products/plasma-2040
+[rp-pico]: https://github.com/rp-rs/rp-hal/tree/main/boards/rp-pico
```

Comment:
Remove reference to pico here
```suggestion
```

Reply:
Actually, update this path to match the git path for pimoroni-plasma-2040 instead.

Reply:
```suggestion
[pimoroni-plasma-2040]: https://github.com/rp-rs/rp-hal/tree/main/boards/pimoroni-plasma-2040
```

Reply:
That makes more sense. That snuck in there from the pimoroni-tiny2040's README which I copied. Fixed.